.github/workflows/main.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
```

Comment:
What is the corresponding GitHub pricing for this?

Reply:
> GitHub Packages is free for public repositories

Reply:
Like Roman said, public repo's are free.